receiver/k8seventsreceiver/README.md

```diff
@@ -212,7 +212,7 @@ spec:
       serviceAccountName: otelcontribcol
       containers:
       - name: otelcontribcol
-        image: otelcontribcol:latest # specify image
+        image: otelcontribcol:latest # If you are not building the Collector locally (make docker-otelcontribcol), specify a published image:   otel/opentelemetry-collector-contrib
```

Comment:
```suggestion
        image: otelcontribcol:latest # This image is created by running `make docker-otelcontribcol`. If you are not building the collector locally, specify a published image: `otel/opentelemetry-collector-contrib`
```
Tiny nit: It may be a bit more clear to explicitly state that the image is created by the command. Feel free to ignore, not a big deal at all.